packages/manager/modules/pci/src/projects/project/components/project-dashboard-activate-banner/project-dashboard-activate-banner.controller.js

```diff
@@ -0,0 +1,5 @@
+export default class PciProjectDashboardActivateBannerController {
+  $onInit() {
+    console.log('PciProjectDashboardActivateBannerController: this: ', this);
+  }
+}
```

Comment:
```suggestion
```
No needed or if it's for anticipate the future you can put a message "todo" for example

Reply:
the controller is removed.